src/main/kotlin/com/toasttab/pulseman/files/FileManagement.kt

```diff
@@ -33,34 +39,41 @@ import javax.swing.filechooser.FileNameExtensionFilter
  * TODO pull all view logic out of here
  */
 object FileManagement {
-    const val appFolderName = "pulseman_config/"
-    const val projectTabsFileName = "project_tabs.json"
+    private val os = System.getProperty("os.name")
 
-    val appFolder = File(appFolderName)
+    // Sand boxed mac apps only have access by default to this folder
+    private val HOME_DIRECTORY = if (os.contains("Mac")) System.getProperty("user.home") + "/Library/Pulseman/" else ""
```

Comment:
Should the `"/Library/Pulseman/"` be a const?

Reply:
Yeah it should, don't know how I missed that one, thanks